tests/perf/perf.js

```diff
@@ -110,9 +110,7 @@
         var str = Array(30 + 1).join('hello world x ') + 'xx!';
         var pos = 5;
 
-        XRegExp.install('natives');
-        var fixedExec = RegExp.prototype.exec;
-        XRegExp.uninstall('natives');
+        var fixedExec = XRegExp.exec;
```

Comment:
The 3 cases below of comparing "shimmed exec" to XRegExp.exec are no longer testing different things. I'd recommend removing the `fixedExec` var entirely and the 3 `.add` calls that use it.

Reply:
Oh, good point! I was essentially doing a naive search/replace on the code, so I didn't notice this. Fixed in https://github.com/slevithan/xregexp/pull/207/commits/9f21bd0c5f0119c3b23b99bc7210061e49887bee